inst/scripts/create_element_table.R

```diff
@@ -0,0 +1,43 @@
+library(xml2)
+
```

Comment:
Would be nice if you could add here a comment mentioning where the "isotopes.xml" file could be downloaded.